packages/visual-editor/src/components/Locator.tsx

```diff
@@ -93,7 +94,7 @@ const TRANSLATIONS = {
     useLocator:
       "Verwenden Sie unseren Locator, um einen Standort in Ihrer Nähe zu finden",
     noResults: "Keine Ergebnisse für dieses Gebiet gefunden",
-    viewMoreInformation: "Mehr Informationen anzeigen",
+    visitPage: "Seite besuchen",
```

Comment:
Didn't you all swap to the new translations stuff? I thought I saw that in some other PRs.